crates/sdk/src/lib.rs

```diff
@@ -185,18 +189,18 @@ impl ProverClient {
     ///
     /// let client = ProverClient::network(private_key, rpc_url, skip_simulation);
     /// ```
+    #[cfg(any(feature = "network", feature = "network-v2"))]
```

Comment:
We need to fix these CI failures before merging. Maybe we need to check sp1-sdk using both the network and network-v2 features ... 

Honestly, if we deprecate network v1 in the next couple months maybe we should just deal with these sins until then.